### PR TITLE
Fix wrong api defination

### PR DESCRIPTION
**Pull request by chilianyi**

### CODE REVIEW

File: api/cluster.proto [hunk 1]
```diff
@@ -382,7 +382,7 @@ service ClusterManager {
 			summary: "update cluster env"
 		};
 		option (google.api.http) = {
-			patch: "/v1/clusters"
+			patch: "/v1/clusters/update"
```
Comment (zheng1) [inline api/cluster.proto]: `post: "/v1/clusters/update_env"`

File: api/cluster.proto [hunk 2]
```diff
@@ -328,7 +328,7 @@ service ClusterManager {
 			summary: "delete clusters"
 		};
 		option (google.api.http) = {
-			delete: "/v1/clusters"
+			delete: "/v1/clusters/delete"
```
Comment (zheng1) [inline api/cluster.proto]: It's best to unify all the action to `post: "/v1/clusters/{action}"`. So this line could be `post: "/v1/clusters/delete`

File: api/cluster.proto [hunk 3]
```diff
@@ -373,7 +373,7 @@ service ClusterManager {
 			summary: "delete cluster nodes"
 		};
 		option (google.api.http) = {
-			delete: "/v1/clusters"
+			delete: "/v1/clusters/delete_nodes"
```
Comment (zheng1) [inline api/cluster.proto]: `post: "/v1/clusters/delete_nodes"`

File: api/cluster.proto [hunk 4]
```diff
@@ -434,7 +434,7 @@ service ClusterManager {
 			summary: "cease clusters"
 		};
 		option (google.api.http) = {
-			delete: "/v1/clusters"
+			delete: "/v1/clusters/cease"
```
Comment (zheng1) [inline api/cluster.proto]: `post: "/v1/clusters/cease"`

Comment (chilianyi) [reply api/cluster.proto]: done

**Pull request merged**